src/pages/SignUp.tsx

```diff
@@ -7,6 +8,7 @@ import useFetch from '../hooks/useFetch';
 /**
  * This component is the page responsible for signing up.
  */
+
```

Comment:
There shouldn't be a newline here. The comment above is a doc block which gives some information on the component below. With this newline, the doc block is lost and isn't tied to the component.

Reply:
Right, my bad